sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/MathExpressionsSuite.scala

```diff
@@ -658,6 +671,62 @@ class MathExpressionsSuite extends SparkFunSuite with ExpressionEvalHelper {
     val intResultsB: Seq[Int] = Seq(314000000, 314200000, 314160000, 314159000, 314159300,
       314159260) ++ Seq.fill(7)(314159265)
 
+    def doubleResultsFloor(i: Int): Any = {
+      val results = Seq(0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 3,
+        3.1, 3.14, 3.141, 3.1415, 3.14159, 3.141592)
+      if (i <= 6) results(i).toLong else results(i)
+    }
+
+    def doubleResultsCeil(i: Int): Any = {
```

Comment:
is it the same as `doubleResults` for `Round`?

Reply:
No, here is the `doubleResults` for `Round` :
`val doubleResults: Seq[Double] = Seq(0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 3.0, 3.1, 3.14, 3.142,
      3.1416, 3.14159, 3.141593)`